app/frontend/components/pages/osbl/show/LocationTab.tsx

```
@@ -0,0 +1,242 @@
+import { NewOsbl } from '@/pages/Contribution/types'
+import 'leaflet/dist/leaflet.css'
+import { MapContainer, TileLayer, Marker, Tooltip } from 'react-leaflet'
+import { Icon, Map, DivIcon } from 'leaflet'
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)